kamelet-support/src/main/java/io/kaoto/backend/model/deployment/kamelet/expression/CSimple.java

```diff
@@ -23,18 +23,18 @@ public CSimple() {
     }
 
     public CSimple(Map map) {
-        if (map.containsKey(EXPRESSION_LABEL)) {
```

Comment:
Are we sure we will always have the key in the map? (I see it works on the tests, but I am not 100% sure)

Reply:
Thinking on using `getOrDefault` instead of `get`

Reply:
`get` is promised to return `null` if there's no entry
https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#get-java.lang.Object-